src/lib/Telemetry/telemetry.cpp

```diff
@@ -144,6 +144,7 @@ void Telemetry::ResetState()
     telemetry_state = TELEMETRY_IDLE;
     currentTelemetryByte = 0;
     currentPayloadIndex = 0;
+    currentQueueIndex = 0;
```

Comment:
This whole thing's a queue! Please use less generic term for this feature, suggest maybe `twoslotLastQueueIndex`?

Reply:
sounds good